data/human/transport missions.txt

```diff
@@ -3085,12 +3091,13 @@ mission "Paradise Fortune 4"
 	on decline
 		payment 525894400
 		"reputation: Republic" <?= -1000
-	
+		log "People" "Diana Howl" `A teenager who stole half a billion credits from her family's wealth, she decided to start a new life outside the paradise planets and you decided to pocket the substantial fortune she left behind to avoid being hunted down.`
```

Comment:
```suggestion
		log "People" "Diana Howl" `A teenager who stole half a billion credits from her family's wealth, she decided to start a new life outside the Paradise Planets and you decided to pocket the substantial fortune she left behind to avoid being hunted down.`
```
Aren't the Paradise Planets capatilized?